.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+on: [push]
+jobs:
+  build:
+    name: Build, lint, and test on Node ${{ matrix.node }} and ${{ matrix.os }}
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node: ['10.x', '12.x', '14.x']
+        os: [ubuntu-latest, windows-latest, macOS-latest]
```

Comment:
Is there any benefit to building across all these environments? You're building for a browser environment, and using node only to build the JS assets. I don't see there being much point in this and we could save a few CPU cycles (the planet needs all the help it can get) by just building using Node 16 (current LTS) and Ubuntu.

Reply:
Yeah, there was no benefit to it, it was the default tsdx build action, I now replaced it with a faster one 🌍